lib/OpenQA/Schema.pm

```diff
@@ -30,7 +30,7 @@ use OpenQA::Utils ();
 
 # after bumping the version please look at the instructions in the docs/Contributing.asciidoc file
 # on what scripts should be run and how
-our $VERSION   = 56;
+our $VERSION   = 57;
```

Comment:
I reduced database number, since i pushed it more then once in the process, so currently it would be 57